docker-compose.yml

```diff
@@ -167,3 +167,5 @@ volumes:
         external: true
     data-search:
         external: true
+    data-s3:
+        external: true
```

Comment:
Moved from the `override` as `initialize-volumes.sh` only reads the main file currently. Means the `unstable` environment will have an empty volume, but 🤷‍♂.